extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -13,84 +13,77 @@
 @ConfigGroup
 public class OidcCommonConfig {
     /**
-     * The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
-     * OIDC discovery endpoint will be called by default by appending a '.well-known/openid-configuration' path to this URL.
-     * Note if you work with Keycloak OIDC server, make sure the base URL is in the following format:
-     * `https://host:port/realms/{realm}` where `{realm}` has to be replaced by the name of the Keycloak realm.
+     * Optional. The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
+     * The OIDC discovery endpoint is called by default by appending a `.well-known/openid-configuration` path to this URL.
+     * For Keycloak, use `https://host:port/realms/{realm}`, replacing `{realm}` with the Keycloak realm name.
      */
     @ConfigItem
     public Optional<String> authServerUrl = Optional.empty();
 
     /**
-     * Enables OIDC discovery.
-     * If the discovery is disabled then the OIDC endpoint URLs must be configured individually.
+     * Optional. Discovery of the OIDC endpoints.
```

Comment:
This changes the way it works. Now it reads as if users have to manually configure the endpoints, as it is not explicitly enabled.
FYI, if you see `defaultValueDocumentation` set to some value then it means that even if the value is Optional, there is some logic there assuming that default doc value. Typically, in such cases, Optional is set to figure out if this property has been explicitly configured by a user or not

Reply:
Thanks. I've dropped `Optional`. Is this what you mean:
```suggestion
     * Discovery of the OIDC endpoints.
```

Reply:
Updated. Thanks.